Jenkinsfile

```diff
@@ -18,7 +15,7 @@ properties([
     pipelineTriggers([pollSCM('H/15 * * * *')])
 ])
 
-node('docker') {
+node('sdk && docker') {
```

Comment:
This seems wrong.  The SDK has nothing to do with this build, since it's just starting a Docker container.  (In fact, it can't be built in the SDK, since the SDK doesn't include Docker.)  This change should be dropped.